pkg/load/transport.go

```diff
@@ -192,8 +238,8 @@ func (r *AttachReader) Read(bs []byte) (int, error) {
 			return 0, err
 		}
 		len := binary.BigEndian.Uint32(bs[4:8])
-		isStderr := bs[0] == 2
-		if isStderr {
+		isFD := bs[0] == byte(r.fd)
+		if !isFD {
```

Comment:
Down the road we could update this attachreader here to print anything from stderr we receive from the proxy container.